roles/cinder-control/tasks/monitoring.yml

```diff
@@ -9,7 +9,7 @@
 
 - name: cinder services check
   sensu_check: name=check-cinder-services plugin=check-cinder-services.sh
-               use_sudo=true
+               use_sudo=true args="-z {{ cinder.sensu_criticality.check_cinder_services }}"
```

Comment:
Why did the argument switch to `-z` here, from `--criticality` in earlier checks?